packages/tooltip/components/Tooltip.tsx

```diff
@@ -58,6 +58,7 @@ class Tooltip extends React.PureComponent<TooltipProps, TooltipState> {
         onFocus={this.handleOpen}
         onBlur={this.handleClose}
         tabIndex={0}
+        data-cy="tooltip"
```

Comment:
I can't see the data-cy for this element.

Reply:
I do
<img width="543" alt="Screen Shot 2019-07-19 at 10 56 07 AM" src="https://user-images.githubusercontent.com/2313998/61544632-dd92d980-aa13-11e9-9470-926a2b0bb234.png">